src/components/TypesOfRecipeCards/CommentCard.jsx

```diff
@@ -0,0 +1,83 @@
+import Pizza from "../../assets/pizza.jpg";
+import User1 from "../../assets/user/User2.svg";
+
+const CommentCard = ({ user, review }) => {
```

Comment:
```suggestion
/* 
 *CommentCard grabs all the reviews created by the user
*/
const CommentCard = ({ user, review }) => {
```
I suggest to move the comment here with multi-line